build/gen_stub.php

```diff
@@ -641,6 +661,12 @@ function parseFunctionLike(
         }
 
         $type = $param->type ? Type::fromNode($param->type) : null;
+        if ($type === null && !isset($docParamTypes[$varName]) && !isset($warnings[$name->getName()])) {
+            $warnings[$name->getName()] = true;
+            //throw new Exception("Missing argument type for function $name()");
+            echo "Warning: Missing argument type for function $name()\n";
```

Comment:
There's way too many missing argument types so let's just have warning until they get fixed. Although probably this PR shouldn't be merged like this till then.

Reply:
How many warnings are we talking here?

Reply:
OMG, it's exactly 293 warnings :/

Reply:
I've fixed quite a few warnings in my 2nd commit. Now, we're around 230. If my 2 resource to object conversion related PRs are also accepted, we can go below 200. If you could have a look at the 2nd commit in the following days (it's perfect after the FF too), I'd like to apply it to master.